### PR TITLE
Update to latest source plugin version.

### PR DESCRIPTION
**Pull request by thetre97**
Have updated to the latest versions of the packages, and also updated the WordpressPost template to the latest queries.

### CODE REVIEW

**Comment by thetre97** [comment]
So, I may have got a little carried away, and added a Netlify Deploy button too.

File: src/templates/WordPressPost.vue [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <template>
   <Layout>
     <h1 v-html="$page.wordPressPost.title"/>
-    <img
+    <g-image
```
[the remark this answers is not in the document]
Comment (thetre97) [reply src/templates/WordPressPost.vue]: Ah, I didn't see that in the docs... thanks for pointing it out!

**Comment by thetre97** [comment]
The reason I added the `netlify.toml` is that I thought it may make the deploy button easier, as it already has the build commands specified. But happy to remove it, and just add a comment in the `README.md` with the build commands?

**Comment by thetre97** [comment]
> And maybe we could have a Deploy to now too? :)

That would be great... but does that support V2? I can't seem to find any docs for it 🤔 
I guess it would if I added a `now.json`? But it would mean adding another file to the repo :)

**Comment by thetre97** [comment]
One other thing, is there a way of adding a default env to make these Netlify checks pass? Using something like https://demo.wp-api.org.

**Comment by hjvedvik** [comment]
Not sure about v2 for now, so we can probably just keep the Netlify button for now :)

We might remove the `netlify.toml` file in other core starters and instead have better documentation on how to use Gridsome with Netlify.

I added the https://demo.wp-api.org URL as an env variable and the PR builds successfully. Just an error about mixed content, so we could find another demo or create our own.

**Pull request merged**